activesupport/lib/active_support/cache.rb

```diff
@@ -186,7 +191,7 @@ def ensure_connection_pool_added!
       # except for <tt>:namespace</tt> which can be used to set the global
       # namespace for the cache.
       def initialize(options = nil)
-        @options = options ? options.dup : {}
+        @options = options ? normalize_options(options.dup) : {}
```

Comment:
```suggestion
        @options = options ? normalize_options(options) : {}
```

Given it is already done inside the method.